scripts/nebula.service

```diff
@@ -119,6 +122,9 @@ function start_daemon {
     local daemon_name=nebula-${1}
     local config=${2}
     local executable=${INSTALL_ROOT}/bin/${daemon_name}
+     if [[ $daemon_name == *-listener ]]; then
+        executable=${INSTALL_ROOT}/bin/${daemon_name%-listener*}
```

Comment:
ditto, nebula-storaged-listener.service don't need to send the para storaged-listener, and this seems redundant.